src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -1033,6 +1138,23 @@ public static Collection<FunctionMetadata> ReadFunctionMetadata(ScriptHostConfig
             // determine the script type based on the primary script file extension
             functionMetadata.ScriptType = ParseScriptType(functionMetadata.ScriptFile);
 
+            JToken isDirect;
+            if (functionConfig.TryGetValue("configurationSource", StringComparison.OrdinalIgnoreCase, out isDirect))
+            {
+                var value = isDirect.ToString().ToLowerInvariant();
+                if (value == "attributes")
+                {
+                    functionMetadata.IsDirect = true;
```

Comment:
I wanted to keep it next to .EntryPoint parsing below . But I can move it. 

---
In reply to: [129905014](https://github.com/Azure/azure-webjobs-sdk-script/pull/1717#discussion_r129905014) [](ancestors = 129905014)